### DISCUSSION TITLE
Help on how to use the GUI

### DISCUSSION THREAD

**Discussion by Coeus1** [The-OpenROAD-Project/OpenROAD#4029] [Q&A] [answered]
Hello,
I am quite new to Openroad and have a question regarding the GUI. I have read that it is possible to invoke the GUI with the command `openroad -gui`. But after entering the command, I get the message `openroad: command not found`.
I have already read the instructions from the documentation, but apparently I am missing something essential.
If anyone has an idea what I am doing wrong, I would appreciate it very much.

(PS: I have the ORFS installed locally on my machine and use _ubuntu 22.04_)

**Comment by rovinski** [comment, marked as answer]
Run `source env.sh`

**Reply by Coeus1** [reply]
Thank you very much
